api/BluetoothRemoteGATTCharacteristic.json

```diff
@@ -630,7 +630,7 @@
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
```

Comment:
I'm not sure this is right or not. Sounds like `deprecated` should win in this case (#15913), what do others think?